src/backend/distributed/executor/multi_server_executor.c

```diff
@@ -36,8 +36,7 @@ int TaskExecutorType = MULTI_EXECUTOR_ADAPTIVE; /* distributed executor type */
 bool BinaryMasterCopyFormat = false; /* copy data from workers in binary format */
```

Comment:
We should be able to remove BinaryMasterCopyFormat after this change

Reply:
Note: We don't generally remove GUCs, because people might not be able to start postgres if they have a non-existent GUC in postgresql.conf. Better to mark it as deprecated in shard_library_init.c